unix_integration/resolver/src/idprovider/interface.rs

```diff
@@ -194,7 +194,8 @@ impl Into<PamAuthResponse> for AuthRequest {
 }
 
 pub enum AuthResult {
-    Success { token: UserToken },
+    Success,
+    SuccessUpdate { token: UserToken },
```

Comment:
Can this be new token so it's clear that we've supplied a new one? I know *you* know it right now, or digging into the source, but it's just one of those DX things.

Reply:
Done!